apps/spruce/src/constants/task.ts

```diff
@@ -221,11 +221,6 @@ export const mapUmbrellaStatusColors: Pick<
     border: purple.dark3,
     text: purple.light3,
   },
-  [TaskStatus.UndispatchedUmbrella]: {
```

Comment:
This change doesn't seem related, was it intentional?

Reply:
This key actually appears in the object twice which threw a type error! check line 209